stories/src/vertical/quote-app.jsx

```diff
@@ -40,7 +40,11 @@ export default class QuoteApp extends Component<Props, State> {
     quotes: this.props.initial,
   };
 
-  onDragStart = (initial: DragStart) => {
+  onDragStart = (initial: DragStart, announce: Announce) => {
```

Comment:
we can remove the announcement stuff from this example and leave it in the new story